icons/semaphoreci.svg

```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Semaphore CI</title><path d="M21.864 5.266a.595.595 0 01.61.257s.467.76.744 1.365c.275.604.542 1.453.542 1.453a.592.592 0 01-.208.628l-1.505 1.073a.757.757 0 00-.284.646s.084.573.096 1.066c.01.485-.04 1.045-.04 1.045a.752.752 0 00.316.628l1.561.993c.192.122.3.398.24.618 0 0-.222.862-.467 1.48-.242.62-.668 1.4-.668 1.4a.592.592 0 01-.595.287l-1.819-.337a.76.76 0 00-.66.245s-.381.489-.754.861c-.414.415-.997.857-.997.857a.76.76 0 00-.254.658l.306 1.824a.592.592 0 01-.298.59s-.79.412-1.414.645c-.623.233-1.488.44-1.488.44a.592.592 0 01-.61-.251l-.967-1.576a.756.756 0 00-.624-.328s-.496.04-.949.025c-.466-.015-.986-.09-.986-.09a.754.754 0 00-.646.281L8.97 23.555a.593.593 0 01-.628.207s-.85-.267-1.456-.543c-.605-.276-1.362-.74-1.362-.74a.592.592 0 01-.256-.61l.385-1.599c.08-.331.338-.796.579-1.037l3.98-3.993a3.7 3.7 0 005.026-5.025l3.988-3.989c.241-.24.705-.5 1.036-.579zM2.134 18.733a.595.595 0 01-.61-.257s-.467-.758-.743-1.364c-.276-.605-.543-1.453-.543-1.453a.592.592 0 01.207-.628l1.506-1.073a.757.757 0 00.283-.646s-.082-.573-.095-1.066c-.01-.485.04-1.046.04-1.046a.752.752 0 00-.316-.628L.303 9.58a.592.592 0 01-.241-.617S.285 8.1.528 7.48c.244-.617.67-1.4.67-1.4a.592.592 0 01.594-.286l1.819.337a.76.76 0 00.661-.244s.38-.489.753-.862c.414-.414.997-.855.997-.855a.76.76 0 00.254-.66L5.97 1.688a.592.592 0 01.298-.591S7.058.685 7.682.45C8.305.219 9.17.012 9.17.012a.592.592 0 01.612.25l.966 1.578a.756.756 0 00.624.326s.496-.039.949-.024c.466.015.986.091.986.091a.754.754 0 00.646-.283L15.028.445a.593.593 0 01.628-.208s.85.268 1.456.544c.605.275 1.362.74 1.362.74a.593.593 0 01.256.61l-.385 1.599a2.59 2.59 0 01-.578 1.036L13.786 8.76a3.7 3.7 0 00-5.026 5.026l-3.988 3.987c-.24.242-.705.5-1.036.58z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Semaphore CI</title><path d="m21.503 13.255-4.158-4.228a3.038 3.038 0 0 0-4.359 0l-3.292 3.347a1.207 1.207 0 0 1-1.721 0l-1.631-1.659L10.5 6.487a6.532 6.532 0 0 1 9.342 0L24 10.715zM8.83 19.479a6.53 6.53 0 0 1-4.671-1.965L0 13.295l2.487-2.529 4.157 4.218a3.056 3.056 0 0 0 4.36 0l3.291-3.348a1.207 1.207 0 0 1 1.721 0l1.631 1.659-4.147 4.229a6.519 6.519 0 0 1-4.671 1.955z"/></svg>
```

Comment:
Thank you @jorgeamadosoria for your contribution!
When comparing with the source file, I'm getting some differences. Here is an example (original in red and you version in black):
![image](https://user-images.githubusercontent.com/12817388/129114673-212d3b23-e1e2-4c1c-9fd8-e47f70eacbbf.png)

Can you take another look at it?

Reply:
I'm happy to take a look at it again, but, huh.... I'm not sure what I'm looking at here, sorry. 
What part of the icon is this?

Reply:
The part shown above is the top right corner, but there are other parts of the icon that seems to be a little bit off. Might be dure to precision level when optimizing the icon.

Reply:
may be.
I'll try again with a different source, see if it gets better.

Reply:
> The part shown above is the top right corner, but there are other parts of the icon that seems to be a little bit off. Might be due to precision level when optimizing the icon.

I've made some changes, perhaps now the fit is better?

Also, can you teach me how you do this comparison? perhaps I can do it on my own prior to pushing and save us all some time.

Reply:
For the comparison, I'm using Inkscape.

I open the original one, grouped every path, resize the icon to 24, center it in a 24x24 viewbox and apply a red color to it.

After this, I drag the contributor SVG over it. Then I apply the closest zoom possible to make sure the icon is exactly the same with the original below and the original above the contributor path. Sometimes, I play with opacity to make sure everything is fine.

Reply:
Hum... Dont know if something went wrong, but now I'm seeing major differences between the source and the updated path. Ex.
![image](https://user-images.githubusercontent.com/12817388/129582250-07588610-9b78-4c44-8d45-a8b1dc67a97a.png)

Reply:
> Hum... Dont know if something went wrong, but now I'm seeing major differences between the source and the updated path. Ex.
> ![image](https://user-images.githubusercontent.com/12817388/129582250-07588610-9b78-4c44-8d45-a8b1dc67a97a.png)

I used a different source this second time around.

I can adjust it on inkscape before pushing again now that I now your process.